README.md

```diff
@@ -80,6 +81,7 @@ export const Wallet: FC = () => {
     const wallets = useMemo(
         () => [
             new PhantomWalletAdapter(),
+            new ExodusWalletAdapter(),
```

Comment:
Order here (and in the other arrays below) can be whatever makes sense.

Reply:
Please remove this change and the changes to the starter projects. We aren't making changes to those at this time, but we actively encourage devs using the project to choose wallets that make sense for their users.